my-code/frontend/src/components/MovieList/style.ts

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-gap: 20px;
+  flex: 1;
```

Comment:
this will make the grid too tall, [moving each row apart](http://drops.significa.co/Screen-Shot-2021-04-09-10-01-17-1617958887)